go/docs/rfcs.go

```diff
@@ -0,0 +1,449 @@
+package docs
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"reflect"
+	"sort"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/PuerkitoBio/goquery"
+	"github.com/sirupsen/logrus"
+	"github.com/vito/booklit"
+)
+
+const resolutionMerge = "resolution/merge"
+const resolutionPostpone = "resolution/postpone"
+const resolutionClose = "resolution/close"
+
+var cachedRFCs []PullRequest
+var cacheOnce = new(sync.Once)
+
+type PullRequest struct {
+	URL     string
+	Number  int
+	Title   string
+	IsDraft bool
+
+	CreatedAt time.Time
+
+	ProposalURL       string
+	QuestionsURL      string
+	OpenQuestionCount int
+
+	Labels    []GitHubLabel
+	Reactions []GitHubReaction
+
+	CommentCount int
+	ReviewCount  int
+}
+
+func (pr PullRequest) ByTotalReactions() int {
+	var s int
+	for _, reaction := range pr.Reactions {
+		s += reaction.Count
+	}
+
+	return s
+}
+
+func (pr PullRequest) ByOpenQuestions() int {
```

Comment:
This and other `By...()` appear to not be used anywhere. I'm fine keeping them, just want to make sure I'm reading it right that they're currently not being used.

Reply:
Yeah, these are here because I experimented with a few other renditions of the table and I wanted to be able to quickly switch between them from the Booklit document:

```tex
\rfcs-table{10}{ByTotalReactions}
```

Happy to remove them if you prefer.